pkg/properties/propertiesGet.go

```diff
@@ -140,12 +143,20 @@ func GetFormatterNamesString(validFormatters map[string]propertiesformatter.Prop
 	formatterNames := strings.Builder{}
 
 	for count, formatterName := range names {
-
 		if count != 0 {
 			formatterNames.WriteString(", ")
 		}
 		formatterNames.WriteString("'" + formatterName + "'")
-	}
 
+	}
 	return formatterNames.String()
+
+}
+
+func checkNameNotUsedWithPrefixSuffixInfix(name string, prefix string, suffix string, infix string) error {
+	var err error
+	if name != "" && (prefix != "" || suffix != "" || infix != "") {
+		err = galasaErrors.NewGalasaError(galasaErrors.GALASA_ERROR_INVALID_PROPERTIES_FLAG_COMBINATION)
+	}
+	return err
```

Comment:
Might be worth trying out cobra's `MarkFlagsMutuallyExclusive`, then we won't need this method. You'll need to update the version of cobra we're using with:
```
go get -u github.com/spf13/cobra@latest
```

Reply:
Thanks, didn't know `MarkFlagsMutuallyExclusive` existed!
I tried it out and since `name` is a persistent flag, declared in a separate file to `suffix` etc, I can't seem to use `MarkFlagsMutuallyExclusive` on them without getting an error that one or the other could not be found. 
If you know a counter solution to this, please do share! Otherwise I will keep it as is, as you're also happy with this.